test/strong_routes/allow_test.rb

```diff
@@ -48,18 +48,29 @@
     end
 
     it "allows access to /users?stuff=12" do
-      get "/users/?stuff=12"
+      get "/users?stuff=12"
+      _(last_response).must_be :ok?
+    end
+
+    it "allows access to /users/profile/anything?stuff=12" do
+      get "/users/profile/anything?stuff=12"
```

Comment:
I had to add this path to the list above to get this to pass after this change. I am guessing this was supposed to check that it doesn't pass though, and was a false positive. Let me know if that's not right.

Reply:
I think this is necessary, and there might have been something similar before I refactored this for 2.0. 🙃